packages/sw-build/src/lib/generate-sw.js

```diff
@@ -55,7 +55,7 @@ const errors = require('./errors');
  * @param {number} [input.maximumFileSizeToCacheInBytes] An optional number to
  * define the maximum file size to consider whether the file should be
  * precached. (Defaults to 2MB).
- * @param {Regex} [input.dontCacheBustUrlsMatching] An optional regex that will
+ * @param {Regexp} [input.dontCacheBustUrlsMatching] An optional regex that will
```

Comment:
I don't know how particular JSDoc is, or whether it even matters in terms of how it generates the docs, but the capitalization of the JavaScript class is [`RegExp`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp). Feel free to ignore if it doesn't matter.